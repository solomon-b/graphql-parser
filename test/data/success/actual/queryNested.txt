Left
    ( Left
        ( Field
            { fieldAlias = Nothing
            , fieldName = "hero"
            , fieldArguments = fromList []
            , fieldDirectives = []
            , fieldSelectionSet = Just
                ( Left
                    ( Left
                        ( Field
                            { fieldAlias = Nothing
                            , fieldName = "name"
                            , fieldArguments = fromList []
                            , fieldDirectives = []
                            , fieldSelectionSet = Nothing
                            }
                        )
                    ) :|
                    [ Left
                        ( Left
                            ( Field
                                { fieldAlias = Nothing
                                , fieldName = "friends"
                                , fieldArguments = fromList []
                                , fieldDirectives = []
                                , fieldSelectionSet = Just
                                    ( Left
                                        ( Left
                                            ( Field
                                                { fieldAlias = Nothing
                                                , fieldName = "name"
                                                , fieldArguments = fromList []
                                                , fieldDirectives = []
                                                , fieldSelectionSet = Nothing
                                                }
                                            )
                                        ) :| []
                                    )
                                }
                            )
                        )
                    ]
                )
            }
        )
    ) :| []