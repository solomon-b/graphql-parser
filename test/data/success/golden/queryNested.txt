[ SelField
    ( Field
        { fieldAlias = Nothing
        , fieldName = "hero"
        , fieldDirectives = []
        , fieldArguments = fromList []
        , fieldSelectionSet =
            [ SelField
                ( Field
                    { fieldAlias = Nothing
                    , fieldName = "name"
                    , fieldDirectives = []
                    , fieldArguments = fromList []
                    , fieldSelectionSet = []
                    }
                )
            , SelField
                ( Field
                    { fieldAlias = Nothing
                    , fieldName = "friends"
                    , fieldDirectives = []
                    , fieldArguments = fromList []
                    , fieldSelectionSet =
                        [ SelField
                            ( Field
                                { fieldAlias = Nothing
                                , fieldName = "name"
                                , fieldDirectives = []
                                , fieldArguments = fromList []
                                , fieldSelectionSet = []
                                }
                            )
                        ]
                    }
                )
            ]
        }
    )
]